mapfishapp/src/main/webapp/app/js/GEOR_wmcbrowser.js

```diff
@@ -418,6 +419,22 @@ GEOR.wmcbrowser = (function() {
             }, view, formPanel]
         });
     };
+    
+    var filterContexts = function(storeData) {
+		for(var i = storeData.length; i--;){
```

Comment:
Can't use Ext.each() because of the inline deletion of an element, that's why I used decrementation instead of incrementation, for allow to remove items in the for loop.